### DISCUSSION TITLE
EDMC-Telemetry plugin

### DISCUSSION THREAD

**Discussion by fasteddy516** [EDCD/EDMarketConnector#1136] [Show and tell]
I've written a plugin that takes dashboard status and journal events and publishes them to an MQTT broker.  The primary use case (why I wrote it and how I use it, at least), is to provide in-game status and feedback to a custom cockpit and controls that respond to in-game events. 

The plugin is available at https://github.com/fasteddy516/EDMC-Telemetry, and I was hoping it could be added to the list of available plugins in the wiki.

**Comment by Athanasius** [comment]
I've finally got around to this.  Added!

**Reply by fasteddy516** [reply]
Wonderful, thank you!
